src/content/docs/accounts/accounts-billing/account-setup/student-program.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: New Relic for Students 
+metaDescription: Sign up for a New Relic student account 
+freshnessValidatedDate: never
```

Comment:
I think you can make this the freshnessValidatedDate today's date. The format is YYYY-MM-DD

Reply:
@paperclypse I set this to `2024-03-01`